services/src/Domain/Interfaces/Repositories/IUserRepository.cs

```diff
@@ -6,9 +6,11 @@ public interface IUserRepository: IBaseRepository<User>
 {
     Task<User?> GetByUsernameAsync(string username);
     Task<User?> GetByEmailAddressAsync(string email);
+    Task<User?> GetByIdAsync(int id);

```

Comment:
It can be removed. There already is method GetAsync(long) that will fetch user by his id.